qa/tasks/cephfs/test_client_recovery.py

```diff
@@ -476,3 +476,23 @@ def _test_fsync(self, dirfsync):
         self.mount_b.mount()
         self.mount_b.wait_until_mounted()
         self.mount_b.run_shell(["ls", "subdir/childfile"])
+
+class TestEvictedClient(CephFSTestCase):
+    MDSS_REQUIRED = 2
+
+    def test_unmount_for_evicted_client(self):
+        """Test if client hangs on unmount after evicting the client."""
+        self.fs.mon_manager.raw_cluster_cmd('fs', 'set', self.fs.name,
+                                            'allow_multimds', 'true',
+                                            '--yes-i-really-mean-it')
+        self.fs.set_max_mds(2)
```

Comment:
Why do we need 2 active MDS?

Reply:
I was following the recipe provided on the tracker, but yeah, it's not required. Fixed.